### DISCUSSION TITLE
Errors with some Tabler icons

### DISCUSSION THREAD

**Discussion by ramoshe** [arthelokyo/astrowind#198] [Q&A]
Hello!
When I try to use some of the Tabler icons I get an error when I try to use them to replace the ones in the template. It seems random which ones, but I'm guessing they might be the newer ones?

Any advice on why this might be or what I can do to update the Taber icon collection?

Thanks in advance!

**Comment by widgeter** [comment]
Hi @ramoshe 

To make it easy to use a variety of free icons, we use the [astro-icon](https://github.com/natemoo-re/astro-icon#readme) dependency. This dependency requires the use of [iconify](https://iconify.design/docs/), which is nothing more than a wrapper for each free icon library.

Sometimes the original icon pack, eg **tabler**, updates its icons, and even [@iconify-json/tabler](https://www.npmjs.com/package/@iconify-json/tabler) hasn't been updated yet.

You can be aware of new @iconify-json/tabler versions or create a new PR on the respective [iconify repository](https://github.com/iconify/iconify/pulls) proposing an update of the library with the new icons.
